2.3/2.3.1/Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM java:8-jre
+
```

Comment:
none exists, there is a 2.3.2 version, but we needed 2.3.1 which is the version jimmy is using.